topics/introduction/tutorials/galaxy-cite/tutorial.md

```diff
@@ -0,0 +1,183 @@
+
+
+> <agenda-title></agenda-title>
+> In this tutorial we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# Citing Galaxy
+Citing Galaxy comes in two main actions: 
+
+1. Citing the global Galaxy Project that provides the core code that all Galaxy instances operate on. 
+2. Citing the instance of Galaxy you did your work on.
+
+## Citing Galaxy Project
+Please cite Galaxy Project’s primary publication: 
+
+- ***The Galaxy platform for accessible, reproducible, and collaborative data analyses: 2024 update***
+  - Nucleic Acids Research, Volume 52, Issue W1, 5 July 2024, Pages W83–W94, [https://doi.org/10.1093/nar/gkae410](https://doi.org/10.1093/nar/gkae410)
+
+This is also found on the Galaxy Project Hub with more specific examples of how to cite individual aspects of Galaxy Project - [https://galaxyproject.org/citing-galaxy/](https://galaxyproject.org/citing-galaxy/)
+
+> <comment-title></comment-title>
+> The main Galaxy Project publication will be included in the list if you follow the instructions below to extract tool citations from your History.
+>
+{: .comment}
+
+## Citing a Galaxy Service
+
+### Service Descriptions
+Each public Galaxy service benefits from being able to track the amazing work done by researchers using its service. Provided below are the DOIs for the Galaxy Australia, France and Main `usegalaxy.*` services. The linked papers are available on Zenodo in the Galaxy Project Zenodo Community at [https://zenodo.org/communities/galaxy/](https://zenodo.org/communities/galaxy/):
+
+- Galaxy Australia - [https://doi.org/10.5281/zenodo.17298823](https://doi.org/10.5281/zenodo.17298823)
+- Galaxy France - [https://doi.org/10.5281/zenodo.17298858](https://doi.org/10.5281/zenodo.17298858)
+- Galaxy Main - [https://doi.org/10.5281/zenodo.17298864](https://doi.org/10.5281/zenodo.17298864)
+
+We recommend including the relevant service name and DOI link in your Methods.
+
+
+### Acknowledging a Service
+Each public Galaxy service is provided through a complex mixture of funding, governance and infrastructure providers. As such, each service provides a cut/paste **Acknowledgement** statement that supports their particular configuration and this can be found on each service. If your publication supports Acknowledgements, we strongly recommend you include the service-recommended **Acknowledgements**.
+
+**Acknowledgement** statements for the four main `usegalaxy.*` services can be found using the following links:
+
+- Galaxy Australia - [https://site.usegalaxy.org.au/about#acknowledgement-statement](https://site.usegalaxy.org.au/about#acknowledgement-statement)
+- Galaxy Europe - [https://usegalaxy-eu.github.io/about](https://usegalaxy-eu.github.io/about)
+- Galaxy France - [https://www.france-bioinformatique.fr/en/services/acknowledgement/](https://www.france-bioinformatique.fr/en/services/acknowledgement/)
+- Galaxy Main - [https://usegalaxy.org/#about](https://usegalaxy.org/#about)
```

Comment:
I do not see a description of what should be added to the Acknowledgment. Is there maybe a better link? Or did I miss something?